astropy/modeling/functional_models.py

```diff
@@ -241,20 +241,22 @@ class Gaussian2D(Fittable2DModel):
         Mean of the Gaussian in x.
     y_mean : float
         Mean of the Gaussian in y.
-    x_stddev : float
-        Standard deviation of the Gaussian in x before rotating by theta.
-        ``x_stddev`` and ``y_stddev`` must be specified unless a covariance
-        matrix (``cov_matrix``) is input.
-    y_stddev : float
-        Standard deviation of the Gaussian in y before rotating by theta.
-        ``x_stddev`` and ``y_stddev`` must be specified unless a covariance
-        matrix (``cov_matrix``) is input.
+    x_stddev : float or None
+        Standard deviation of the Gaussian in x before rotating by theta. Must
+        be None if a covariance matrix (``cov_matrix``) is provided. If no
+        ``cov_matrix`` is given, ``None`` means the default value (1).
```

Comment:
Instead of \`\`None\`\`, of course you could do simply \`None\`, which will link to python docs.

Same for `float or None` for the `x_stddev` parameter type  and all None's below for `y_stddev`.